src/components/forms/EditAssignmentForm/EditAssignmentForm.js

```diff
@@ -407,7 +411,7 @@ class EditAssignmentForm extends Component {
 
           <Grid fluid>
             <Row>
-              <Col md={5}>
+              <Col md={6}>
```

Comment:
Do not touch my carefully calibrated col sizes.

Reply:
Too bad, this is better...